Package.swift

```diff
@@ -27,6 +27,7 @@ let package = Package(
     products: [
         .executable(name: "swift-openapi-generator", targets: ["swift-openapi-generator"]),
         .plugin(name: "OpenAPIGenerator", targets: ["OpenAPIGenerator"]),
+        .plugin(name: "OpenAPIGeneratorCommandPlugin", targets: ["OpenAPIGeneratorCommandPlugin"]),
```

Comment:
```suggestion
        .plugin(name: "OpenAPIGeneratorCommand", targets: ["OpenAPIGeneratorCommand"]),
```

I think `OpenAPIGeneratorCommand` is a good name. `Plugin` is redundant, as the build system ensures that this product type is used in the correct context. It also reads better when it shows up in Xcode, as you're "invoking the OpenAPI generator command", and a command is a type of a plugin, so no need to use the phrase `...CommandPlugin`.

Reply:
I agree, but FYI, ironically there is a diagnosis bug that makes it so you can still declare an extension in the "dependencies" array of targets: https://github.com/apple/swift-package-manager/issues/6676
Luckily for command plugins, you don't even need to explicitly declare the plugin target as a dependency.

Reply:
Understood. I think it's okay to skate to where the puck will be once known-and-reported issues in the tools are fixed 🙂